.eslintrc.cjs

```diff
@@ -0,0 +1,11 @@
+require('@rushstack/eslint-patch/modern-module-resolution');
+
+module.exports = {
+  root: true,
+  extends: [
+    'plugin:vue/vue3-essential',
+    '@vue/eslint-config-airbnb-with-typescript',
+    'prettier', // Turns off the formatting rules from the linter since formatting is handled by prettier
+  ],
+  parser: 'vue-eslint-parser',
+};
```

Comment:
Nitpick: Does this need to be in common-js or can we do es modules to make things a tad more uniorm?

Reply:
According to their doc, eslint doesn't support ES module at this time